hyperdbg/include/SDK/Headers/RequestStructures.h

```diff
@@ -266,12 +266,13 @@ typedef struct _DEBUGGER_FLUSH_LOGGING_BUFFERS
  */
 typedef enum _DEBUGGER_TEST_QUERY_STATE
 {
-    TEST_QUERY_HALTING_CORE_STATUS     = 1, // Query constant to show detail of halting of core
-    TEST_QUERY_PREALLOCATED_POOL_STATE = 2, // Query pre-allocated pool state
-    TEST_QUERY_TRAP_STATE              = 3, // Query trap state
-    TEST_BREAKPOINT_TURN_OFF_BPS       = 4, // Turn off the breakpoints
-    TEST_BREAKPOINT_TURN_ON_BPS        = 5, // Turn on the breakpoints
-
+    TEST_QUERY_HALTING_CORE_STATUS                                          = 1, // Query constant to show detail of halting of core
+    TEST_QUERY_PREALLOCATED_POOL_STATE                                      = 2, // Query pre-allocated pool state
+    TEST_QUERY_TRAP_STATE                                                   = 3, // Query trap state
+    TEST_BREAKPOINT_TURN_OFF_BPS                                            = 4, // Turn off the breakpoints
+    TEST_BREAKPOINT_TURN_ON_BPS                                             = 5, // Turn on the breakpoints
+    TEST_BREAKPOINT_TURN_OFF_BPS_ANS_EVENTS_FOR_COMMANDS_IN_REMOTE_COMPUTER = 6, // Turn off the breakpoints and events for executing the commands in the remote computer
```

Comment:
'BPS_ANS_EVENTS' did you mean 'BPS_AND_EVENTS'?

Reply:
Good catch, thanks to both of you